docs/analyst_guide.md

```diff
@@ -0,0 +1,217 @@
+# Access levels
```

Comment:
Although not a long document, might be helpful to put a table of contents above this.

Reply:
Also might put a link to the notebook at the beginning as a source of examples etc.